tests/test_git.py

```diff
@@ -0,0 +1,13 @@
+"""
+Unit-tests for git
+"""
+
+import commodore.git as git
+
+
+def test_reconstruct_api_response(tmp_path):
```

Comment:
This is Python >= 3.6, correct? In this case I'd suggest:

```suggestion
def test_reconstruct_api_response(tmp_path: Path):
```

Doesn't really make the code safer, but we could run a linter over it to ensure that `tmp_path` is not any other type of object than `pathlib.Path`.